tsuru/client/apps.go

```diff
@@ -590,24 +629,53 @@ func ShortID(id string) string {
 	return id
 }
 
-func (a *app) String() string {
-	format := `{{ if .Error -}}
+const simplifiedFormat = `{{ if .Error -}}
 Error: {{ .Error }}
 {{ end -}}
 Application: {{.Name}}
-Description:{{if .Description}} {{.Description}}{{end}}
-Tags:{{if .TagList}} {{.TagList}}{{end}}
+{{- if .Description }}
+Description: {{.Description}}
+{{- end }}
+{{- if .TagList }}
+Tags: {{.TagList}}
+Created by: {{.Owner}}
+{{- end }}
+Platform: {{.Platform}}
+Plan: {{ .Plan.Name }}
+Pool: {{.Pool}} ({{ .Provisioner }}{{ if .Cluster}} cluster: {{ .Cluster }}{{end}})
+{{if not .Routers -}}
+Router:{{if .Router}} {{.Router}}{{if .RouterOpts}} ({{.GetRouterOpts}}){{end}}{{end}}
+{{end -}}
+Teams: {{.TeamList}}
+{{- if .InternalAddr }}
+Cluster Internal Addresses: {{.InternalAddr}}
+{{- end }}
+Cluster External Addresses: {{.Addr}}
+{{- if .SimpleServicesView }}
+Bound Services: {{ .SimpleServicesView }}
+{{- end }}
+`
+
+const fullFormat = `{{ if .Error -}}
+Error: {{ .Error }}
+{{ end -}}
+Application: {{.Name}}
+{{- if .Description }}
```

Comment:
I would keep the old behavior in this one (showing empty description), as this is the fullFormat.
I find it more explicit than omitting. 

What do you think? 
(the same for the rest of the template)

Reply:
Also, this change forces rewriting some tests, which is considered breaking the contract for some (like myself, sometimes 😄 )